experimental/Experimental.jl

```diff
@@ -16,7 +16,6 @@ const oldexppkgs = [
 # DO NOT USE THIS UNLESS YOU KNOW THE CONSEQUENCES.
 # For more background, see https://github.com/oscar-system/Oscar.jl/issues/2300.
 const orderedpkgs = [
-  "JuLie",
   "IntersectionTheory",
   "OrthogonalDiscriminants",  # needs code from JuLie
```

Comment:
If they just depend on JuLie (and not on one another), they can be removed here. (I didn't check, but assume from the comment)

Reply:
I also assumed so, but did not check. I'll check and adjust this.